charts/nsq/templates/_helpers.tpl

```diff
@@ -61,20 +61,23 @@ nsqlookupd selector labels
 */}}
 {{- define "nsq.nsqlookupd.selectorLabels" -}}
 app.kubernetes.io/component: nsqlookupd
+app.kubernetes.io/instance: {{ .Release.Name }}
```

Comment:
It looks like this is already in `nsq.commonLabels` just above; is that not being used correctly for some deployments or pods or something?

Reply:
ohh I see ... (I have used k8s, but with a simpler scheme for labels, without this kind of possibility)